src/client/components/ErrorModal.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { WarningTriangle } from './icons/WarningTriangle'
+import { Modal, ModalProps } from './ModalNew'
+
+type Props = ModalProps & {
+  children: React.ReactNode
+}
+
+const Container = styled.div`
+  padding: 3rem;
+  padding-left: 5rem;
+  position: relative;
+
+  svg {
+    position: absolute;
+    left: 2.5rem;
+    top: 3.25rem;
+  }
```

Comment:
This is for `WarningTriangle`, right? But it will target any `svg` inside the container.

I would consider wrapping only the element we want to target:

```
const StyledWarningTriangle = styled(WarningTriangle)`
  position: absolute;
  left: 2.5rem;
  top: 3.25rem;
`
```

Reply:
Good thinking! :D